cmd/serverless-init/log/log.go

```diff
@@ -92,11 +96,48 @@ func SetupLog(conf *Config, tags map[string]string) {
 }
 
 func (cw *CustomWriter) Write(p []byte) (n int, err error) {
-	fmt.Print(string(p))
-	Write(cw.LogConfig, p, cw.IsError)
+	if len(p) > maxBufferSize {
+		p = p[:maxBufferSize]
+	}
```

Comment:
what happens if a log is >256Kb?

Reply:
I was unclear if the 256kb mentioned in [the docs](https://docs.datadoghq.com/agent/logs/log_transport/?tab=https) was pre or post compression, though I think post makes the most sense.

I wasn't sure what would be the appropriate max for this, though we should definitely have something. This logic just truncates the log to the max size